package.json

```diff
@@ -61,7 +61,7 @@
     "cors": "^2.8.5",
     "country-data": "0.0.31",
     "express": "^4.17.1",
-    "filenamify": "^4.3.0",
+    "filenamify": "^5.0.0",
```

Comment:
filenamify v5 is now pure ESM. freyr can't require it.

https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

Reply:
freyr now supports esm. https://github.com/miraclx/freyr-js/pull/202